packages/aws-cdk-lib/core/lib/custom-resource-provider/cross-region-export-providers/export-writer-provider.ts

```diff
@@ -54,22 +55,31 @@ export class ExportWriter extends Construct {
       });
   }
   private readonly _references: CrossRegionExports = {};
+  private readonly provider: CustomResourceProvider;
+  private readonly resourceArns = new Set<string>;
   constructor(scope: Construct, id: string, props: ExportWriterProps) {
     super(scope, id);
     const stack = Stack.of(this);
     const region = props.region ?? stack.region;
+    this.resourceArns.add(
+      stack.formatArn({
+        service: 'ssm',
+        resource: 'parameter',
+        region,
```

Comment:
Do we want to add this arn, even if it's unresolved here? If not, we can call `this.addRegionToPolicy()`.